MANIFEST.in

```diff
@@ -0,0 +1,3 @@
+include mmfunctions/*.py
```

Comment:
may I know the reason behind this function line?

Reply:
this is a bug. thanks for pointing it out